coordinator/internal/logic/auth/login.go

```diff
@@ -106,6 +106,12 @@ func (l *LoginLogic) Check(login *types.LoginParameter) error {
 			}
 		}
 	}
+
+	if login.Message.ProverProviderType != types.ProverProviderTypeInternal && login.Message.ProverProviderType != types.ProverProviderTypeExternal {
```

Comment:
need to consider the darwinV2 version, the prover doesn't have a `ProverProviderTypeKey`, need to set a default value